frontend/taipy-gui/src/components/Taipy/Selector.tsx

```diff
@@ -424,10 +453,56 @@ const Selector = (props: SelectorProps) => {
         },
         [dispatch, updateVarName, propagate, lovVarName, valueById, props.onChange, module]
     );
+    const handleCheckAllAutoChange = useCallback(
+        (event: SelectChangeEvent<HTMLInputElement>, checked: boolean) => {
+            const sel = checked ? lovList.slice() : [];
+            const ids = sel.map((elt) => elt.id);
+            setAutoValue(sel);
+            setSelectedValue(ids);
+            dispatch(
+                createSendUpdateAction(
+                    updateVarName,
+                    ids,
+                    module,
+                    props.onChange,
+                    propagate,
+                    valueById ? undefined : lovVarName
+                )
+            );
+        },
+        [lovList, dispatch, updateVarName, propagate, lovVarName, valueById, props.onChange, module]
+    );
+    const renderAutoInput = useCallback(
```

Comment:
Would it make sense to rename this to 'AutoInput' for homogeneity with component names?

Reply:
This is a function that returns a component but a component :)